docs/overview/changelog.md

```diff
@@ -53,7 +63,8 @@ log][commit-log] for an up-to-date list of all changes.
   the user experience.
 - The LDAP integration now supports syncing user groups.
 - The SCIM integration now supports syncing user roles.
-- Added support for sorting the recent view by unread count.
+- The recent view now displays the date range it is displaying,
+  supports fetching more conversations and sorting by unread count.
 - Added support for printing a message feed as a lightweight
```

Comment:
Maybe:

The recent view now indicates the date range it is displaying, and supports fetching more conversations and sorting by unread count.